### ISSUE TITLE
2023 reviews?

### ISSUE THREAD

**Issue by mjpost** [nlposs/NLP-OSS#42]
Hi, I just received acceptance notifications for papers, but there are no reviews visible. Is this by mistake?

**Comment by alvations** [comment]
Hmmm, might be the review setup. Let me try again to double check on the openreview setups.

**Comment by alvations** [comment]
Could you check again with a browser refresh / logout + login to see if the reviews show up? 

Apologies for the confusion.

**Comment by mjpost** [comment]
Thanks. Reloaded / logged out and back in / tried a different browser, still showing zero reviews for both my papers. For reference:

![image](https://github.com/nlposs/NLP-OSS/assets/455256/ae8f45db-10ba-4be0-9430-a52e45cc87de)

**Comment by alvations** [comment]
Thanks for checking on the reviews! We're contacting OpenReview support to check if there's anything wrong on the setup. 

We'll give an update once we hear back from them.

**Comment by mjpost** [comment]
Thanks, @alvations. Strange. Still no reviews visible.

**Comment by alvations** [comment]
@mjpost The @openreview folks have rectify the issue on their end and from our system, it looks like the authors can now read the reviews. Please let us know if the reviews are still not appearing on your end.

Thank you again for raising the issue!

**Comment by mjpost** [comment]
Thanks very much!
